docs/README.md

```diff
@@ -57,4 +57,45 @@ Corresponding excerpt from package.json:
   "lint-staged": {
     "*.{js,ts}": "eslint --cache --fix"
   }
-```
\ No newline at end of file
+```
+
+## Conventional commits:
+
+> __Motivation:__
+> 
+> Using conventional commits helps standardize the format of commit messages and allows automatic generation of [CHANGELOG.md](../CHANGELOG.md) file.
```

Comment:
This is great! It would be great if you can do a demo to the team sometime so we're on the same page

Reply:
I will do, in the docs, it is also explained a little ;-)

Reply:
Yea a demo would be great.